rust/azure_iot_operations_protocol/src/rpc_command/executor.rs

```diff
@@ -751,8 +763,8 @@ where
                         }
                         rt
                     } else {
-                        log::error!(
-                            "[{}][pkid: {}] Response topic missing",
+                        log::warn!(
```

Comment:
Isn't this a protocol violation?

Reply:
yes (aio protocol violation), but so are most of the other things we validate. Since this doesn't stop the executor completely, just ignores one message, this should be warn. The non-error scenario for this is a message on the same topic filter that isn't a command request at all